cms/__init__.py

```diff
@@ -1,2 +1 @@
-__version__ = '4.1.1'
-
+__version__ = "4.1.5dev"
```

Comment:
```suggestion
__version__ = "4.2.0dev"
```
We're closing on on 4.2...

Reply:
On second thoughts, this does not work, since the version is part of all asset paths (js and css). :-(